doc/book/src/installation.md

```diff
@@ -1,30 +1,34 @@
 # Installation
 
+## Requirements
+
+Maelstrom currently only supports Linux.
+
+...presumably there are also kernel version requirements?
```

Comment:
Any other requirements?

Reply:
Yeah, we should document our requirements more clearly... Part of the problem is we don't know exactly. :-)

#339